src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
I'd like to leave this in place because it filters out the extension targets, and sometimes we can't positively identify the page target and have to just pick. But in your example, looks like `target.type === 'app'`, so how about adding || that? You don't need to attach to the background_script one right?